card/mod/01_history/set/all/actions.rb

```diff
@@ -0,0 +1,47 @@
+# -*- encoding : utf-8 -*-
```

Comment:
Ok, so the ordering is purely for getting the load order right when they are included into the Card class?

My request is that we address the selection of modules at the same time we handle ordering.  I wan't to be able to skip some of the modules in the gem (card) with configuration overrides, and I think it would be fine to just override a default configuration the also gets us the load ordering.